packages/types/src/transaction.ts

```diff
@@ -169,3 +175,17 @@ export type TransactionSource =
   | 'component'
   /** Name of a background task (e.g. a Celery task) */
   | 'task';
+
+/**
+ * Object representing metadata about when a transaction name was changed.
+ */
+export interface TransactionNameChange {
+  // new name
+  name: string;
+  // unix timestamp when the name was changed
+  timestamp: number;
```

Comment:
FTR: The develop spec and Relay protocol definition should use the same type as other timestamps in the protocol, allowing for ISO strings in addition to numbers.